python_modules/dagster/dagster/_core/definitions/auto_materialize_rule.py

```diff
@@ -297,6 +297,21 @@ def skip_on_required_but_nonexistent_parents() -> "SkipOnRequiredButNonexistentP
         """
         return SkipOnRequiredButNonexistentParentsRule()
 
+    @public
+    @staticmethod
+    def skip_on_backfill_in_progress(
+        skip_all_partitions_of_backfilling_asset: bool = False,
+    ) -> "SkipOnBackfillInProgressRule":
+        """Skip an asset's partitions if targeted by an in-progress backfill.
+
+        Attributes:
+            skip_all_partitions_of_backfilling_asset (Optional[bool]): If true, skips
+                all partitions of the backfilling asset, regardless of whether the partition
```

Comment:
nit:

If True, skips all partitions of the asset being backfilled, regardless of whether the specific partition is targeted by a backfill. If False, skips only partitions targeted by a backfill.

Defaults to False